analysis_scripts/analyze_quote_data/location-analysis_all_years.md

```diff
@@ -0,0 +1,234 @@
+    ## if this guess is not correct, or fix the file write command that created the
+    ## file to create a valid file.
+
+    ## Warning in fread(country_file): Detected 12 column names but the data has 13
+    ## columns (i.e. invalid file). Added 1 extra default column name for the first
+    ## column which is guessed to be row names or an index. Use setnames() afterwards
+    ## if this guess is not correct, or fix the file write command that created the
+    ## file to create a valid file.
+
+    ## Warning in fread(country_file): Detected 12 column names but the data has 13
+    ## columns (i.e. invalid file). Added 1 extra default column name for the first
+    ## column which is guessed to be row names or an index. Use setnames() afterwards
+    ## if this guess is not correct, or fix the file write command that created the
+    ## file to create a valid file.
+
+    ## Warning in fread(country_file): Detected 12 column names but the data has 13
+    ## columns (i.e. invalid file). Added 1 extra default column name for the first
+    ## column which is guessed to be row names or an index. Use setnames() afterwards
+    ## if this guess is not correct, or fix the file write command that created the
+    ## file to create a valid file.
+
+``` r
+full_loc_df = full_loc_df[-1,]
+
+full_loc_df = subset(full_loc_df, est_un_region != "" & 
+                                        est_un_subregion != "" &
+                                        est_un_region != "NO_EST" & 
+                                        est_un_subregion != "NO_EST")
+
+head(full_loc_df)
+```
+
+    ##   est_country_code            file_id                              text
+    ## 2               ae news.2010.647.html rochester institute of technology
+    ## 3               ae news.2010.491.html rochester institute of technology
+    ## 4               ae       468609a.html                               ias
+    ## 5               af       464014b.html                       afghanistan
+    ## 6               af news.2010.684.html                       afghanistan
+    ## 7               af       465990a.html                       afghanistan
+    ##            ner          est_country est_un_region est_un_subregion year
+    ## 2 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 3 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 4 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 5      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+    ## 6      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+    ## 7      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+
+Now we join the tables together for comparison.
+
+``` r
+full_loc_df$is_benchmark = "full_data"
+bm_loc_df$is_benchmark = "benchmark"
+
+bm_full_loc_df = rbind(subset(full_loc_df, year %in% c("2010", "2015", "2020")), 
```

Comment:
Yes, this is correct. For the comparison with the benchmark (bm_full_loc_df) I only compare on those 3 years, since I only have those 3 years in the benchmark (bm_loc_df).
The reason I do this is because I refer to the full dataframe later (full_loc_df) when I analyze the location data itself, instead of only comparing with the benchmark in the section **`Compare locations over all years`**